src/OrchardCore.Modules/OrchardCore.Microsoft.Authentication/Deployment/AzureADDeploymentSource.cs

```diff
@@ -21,8 +21,8 @@ protected override async Task ProcessAsync(AzureADDeploymentStep step, Deploymen
 
         result.Steps.Add(new JsonObject
         {
-            ["name"] = "Settings",
-            ["AzureADSettings"] = JObject.FromObject(azureADSettings),
+            ["name"] = nameof(AzureADSettings),
```

Comment:
don't use nameof() and keep it string like we do everywhere else. because if the class name change, this should not change otherwise it would break people

Reply:
Ok